### PR TITLE
Replace boolean with bool in service.md.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
Bug fix: replace `boolean` with `bool`.

### CODE REVIEW

File: runtime/vm/service/service.md [hunk 1]
```diff
@@ -615,8 +615,8 @@ reference will be returned.
 
 ```
 AllocationProfile getAllocationProfile(string isolateId,
-                                       boolean reset [optional],
-                                       boolean gc [optional])
+                                       bool reset [optional],
```
Comment (bkonyi) [inline runtime/vm/service/service.md]: Thanks! Can you make the same changes in `service_dev.md` as well?

Comment (kenzieschmoll) [reply runtime/vm/service/service.md]: done 👍

**Review by bkonyi — APPROVED** [review]

**Comment by kenzieschmoll** [comment]
Since merging is blocked, @bkonyi will push these changes through.

**Comment by mraleph** [comment]
The actual mergable change is here: https://dart-review.googlesource.com/c/sdk/+/106960

**Comment by bkonyi** [comment]
Merged here: https://dart-review.googlesource.com/c/sdk/+/106983

**Pull request closed without merging**